lib/commands.js

```diff
@@ -273,10 +274,26 @@ var operations = module.exports.operations = {
       context.newTemplate = templateBody;
       context.oldTemplate = oldTemplate;
       context.oldParameters = oldParams;
+
       context.next();
     });
   },
 
+  getMasterConfig: function(context) {
+    if (context.overrides.masterConfig) {
+      var getMasterConfig = d3.queue();
+      var s3Url = context.overrides.masterConfig;
+      getMasterConfig.defer(lookup.defaultConfiguration, s3Url);
+      getMasterConfig.await(function(err, masterConfigJSON) {
+        Object.keys(masterConfigJSON).forEach(function(key) {
+          if (context.oldParameters[key])
+            context.oldParameters[key] = masterConfigJSON[key];
+        });
+        context.next();
+      });
+    } else context.next();
```

Comment:
The tests never take this `else` pathway. Could you add one simple test that shows that `.operations.getMasterConfig` no-ops when there's no `overrides.masterConfig`?